cmd/upgrade/roles/cmd.go

```diff
@@ -919,6 +923,14 @@ func buildOperatorRoleCommandsFromCluster(
 		)
 		_, err = awsClient.IsPolicyExists(policyARN)
 		hasPolicy := err == nil
+		policyAttached := false
```

Comment:
Can policyAttached by renamed to something like
RequiredRolePolicyAttached?
The difference between HasPolicy and PolicyAttached isn't clear for me. 😢